api/desecapi/tests/test_user_management.py

```diff
@@ -900,21 +900,30 @@ def assertRenewDomainVerificationSuccessResponse(self, response):
             status_code=status.HTTP_200_OK
         )
 
-    def test_renew_domain_non_local_public_child(self):
-        user = User.objects.get(email=self.email)
-        domain = self.create_domain(owner=user)
+    def _test_renew_domain_immortal(self, domain):
         for days in [182, 184]:
             domain.published = timezone.now() - timedelta(days=days)
             domain.renewal_changed = domain.published
             domain.save()
 
-            self.assertEqual(Domain.objects.get(pk=domain.pk).renewal_state, Domain.RenewalState.FRESH)
```

Comment:
test for FRESH not needed anymore?

Reply:
refactored tests to have a DYN class and a non-DYN class. In each class, there's a test that FRESH domains only reach the next scavenging step when older than 182 days (compare `test_renew_domain_fresh_182_days()` vs `test_renew_domain_fresh_183_days()`). I think that covers it.